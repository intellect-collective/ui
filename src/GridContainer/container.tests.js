import React from 'react';
import {
    Container
} from '../..';

describe('Container', () => {
    it('Can display', () => {
        const wrapper = mount(
            <Container>test</Container>
        );
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('.container').exists()).toBe(true);
    });
    it('Can change classname', () => {
        const wrapper = mount(<Container className="my-container" />);
        expect(wrapper.find('.my-container').exists()).toBe(true);
    });
});
