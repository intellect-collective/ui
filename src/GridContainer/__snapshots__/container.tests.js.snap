// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Can display 1`] = `
<Container>
  <div
    className="container"
  >
    test
  </div>
</Container>
`;
